recipe/meta.yaml

```diff
@@ -33,7 +33,7 @@ requirements:
     - packaging
     - pandas
     - panel >=0.11.0
-    - param >=1.9.0
+    - param >=1.9.0,<3.0
```

Comment:
if we update run section for the same package version we need to bump up build number